oeqLinkMigrationLTI/src/main/java/org/apereo/openequella/integration/blackboard/linkmigrationlti/FixerUtils.java

```diff
@@ -421,15 +421,55 @@ private void setNameAndDescription(FixerResponse resp, String textToSearch, int
 	}
   }
 
-  private String removeOldIntegLink(String str, int level, String method) {
+  private String scrubDescription(String str, int level, String method, String[] urls) {
     final String start = "<td><table";
     final String end = "</table></td></tr>";
-     final String toRemove = StringUtils.substringBetween(str, start, end);
-     if(toRemove.contains("href=\"/webapps/dych-tle-")) {
-       return StringUtils.replace(str,start+toRemove+end, "<td/></tr>");
-	 } else {
-       log(level+1, "Not removing old integration link from the body - could not find the 'dych-tle' table");
-       return str;
+     final String[] toRemovePossibilities = StringUtils.substringsBetween(str, start, end);
+     for(String toRemove : toRemovePossibilities) {
+	   if (toRemove.contains("href=\"/webapps/dych-tle-")) {
```

Comment:
indentation seems off here, the `if` appears to be inside `for` but is not indented as such

Reply:
Opened #37 to resolve formatting issues repo-wide.